aesara/link/jax/dispatch.py

```diff
@@ -956,20 +956,11 @@ def bartlett(x):
 
 @jax_funcify.register(FillDiagonal)
 def jax_funcify_FillDiagonal(op, **kwargs):
+    def f(value, diagonal):
```

Comment:
```suggestion
    def filldiagonal(value, diagonal):
```

These function names can be useful when debugging.